docs/src/assets/scss/components/resources.scss

```diff
@@ -47,7 +47,6 @@
         left: 0;
         offset-inline-start: 0;
         top: 0;
-        block-inline-start: 0;
```

Comment:
This was probably supposed to be `offset-block-start`, like in this `&::after` which otherwise looks the same:

https://github.com/eslint/eslint/blob/d2cecb4ad2a6d33444cf0288a863c43acb3b468a/docs/src/assets/scss/components/rules.scss#L107-L117

Reply:
Good call, added that version instead